hironx_ros_bridge/src/hironx_ros_bridge/hironx_dashboard.py

```diff
@@ -34,7 +34,11 @@
 
 import os
 
-from python_qt_binding.QtGui import QMessageBox, QLabel, QPalette
+try: # Qt 4
+    from python_qt_binding.QtWidgets import QMessageBox, QLabel, QPalette
```

Comment:
This line works with Qt4, but I think you should use `QtGui` instead of `QtWidgets` because original code uses `QtGui`